UniAtHome/UniAtHome.BLL/Util/MappingProfile.cs

```diff
@@ -0,0 +1,15 @@
+﻿using AutoMapper;
+using UniAtHome.BLL.DTOs;
+using UniAtHome.DAL.Entities;
+
+namespace UniAtHome.BLL.Util
```

Comment:
Don't name folders as Util, Extensions etc: it does not make any sense. Give meaningful names